pkg/backend/aws/ecs.go

```diff
@@ -216,12 +211,12 @@ func (ab *Backend) getNetworkConfig() *ecstypes.NetworkConfiguration {
 func (ab *Backend) Logs(ctx context.Context, stackName string, serviceName string, since string) error {
 	defer diagnostics.AddProfilerRuntime(ctx, time.Now(), "Logs")
 	endTime := time.Now()
+	var startTime *int64
 
 	duration, err := time.ParseDuration(since)
```

Comment:
We should probably handle the error here with a message of something like "that time format isn't supported"